File: Year 2/Semester 4/MPP/JavaFifthWeek/src/main/java/com/example/temasaptamana5/repository/interfaces/IArtistRepository.java
package com.example.temasaptamana5.repository.interfaces;

import com.example.temasaptamana5.domain.Entity;
import com.example.temasaptamana5.repository.IRepository;

public interface IArtistRepository <ID, E extends Entity<ID>> extends IRepository<ID, E> {
}

File: Year 2/Semester 4/Programming and Projection Environments/Week 9/ProtobuffProjectCSharp/persistence/interfaces/IArtistRepository.cs
﻿using ProtobuffProject.domain;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ProtobuffProject.repository.interfaces
{
    public interface IArtistRepository : IRepository<int, Artist<int>>
    {
    }
}